mrchecker-framework-modules/mrchecker-core-module/src/main/java/com/capgemini/mrchecker/test/core/logger/BFLoggerInstance.java

```diff
@@ -4,37 +4,30 @@
 import java.io.FileNotFoundException;
 import java.io.IOException;
 import java.io.PrintWriter;
-import java.nio.file.Paths;
 import java.util.Arrays;
 
-import org.apache.log4j.Appender;
-import org.apache.log4j.AsyncAppender;
-import org.apache.log4j.ConsoleAppender;
-import org.apache.log4j.FileAppender;
-import org.apache.log4j.Level;
-import org.apache.log4j.Logger;
-import org.apache.log4j.PatternLayout;
+import org.apache.log4j.*;
```

Comment:
is it right to import * instead of single ones?

Reply:
In eclipse configuration there is settings Number of imports needed for .* - default 99
In InteliJ I found settings: Class count to use import with .* - default 3 and 5

Let's discuss it on daily mtg.

Reply:
I changed IntelliJ settings and replaced all .* with explicit imports.